src/osx/carbon/cursor.cpp

```diff
@@ -17,7 +17,9 @@
     #include "wx/image.h"
 #endif // WX_PRECOMP
 
+#include "wx/filename.h"
 #include "wx/xpmdecod.h"
+#include "wx/stdpaths.h"
```

Comment:
Very minor, but could this line go above, please, to keep the header in alphabetical order?